client/tailwind.config.js

```diff
@@ -56,7 +56,16 @@ module.exports = {
       LinkText: 'LinkText',
       ButtonText: 'ButtonText',
     },
-    fontFamily,
+    /** Revise the theme, but not the raw tokens to use the custom property but fallback
+     * to the original value if the custom property is malformed or somehow not provided.
+     */
+    fontFamily: Object.entries(fontFamily).reduce(
+      (obj, [key, value]) => ({
+        ...obj,
+        [key]: `var(--w-font-${key}, ${value.join(', ')})`,
+      }),
+      {},
+    ),
```

Comment:
Have you considered hard-coding this instead? It means a tiny bit more work should we introduce a new font family, but it’s much clearer what is going on for people who don’t know JS / Tailwind that well.

```suggestion
    fontFamily: {
      sans: 'var(--w-font-sans)',
      mono: 'var(--w-font-mono)',
    },
```

I’m also not convinced we do want this fallback. To me it feels like if the custom property is malformed it’d be much simpler to troubleshoot if we weren’t defaulting to a fallback. Same if it was absent, which feels very unlikely.

Reply:
yeah - I agree on both counts, thanks for the feedback.
Will fix up.